e2e/helpers/wallet.helper.ts

```diff
@@ -157,4 +198,84 @@ export class WalletHelper {
 
         return { transaction, txid, blockhash };
     }
+
+    private createInputFromUtxo(utxo: UTXO, keyPair: BIP32Interface): any {
+        const input: any = {
```

Comment:
```suggestion
    private createInputFromUtxo(
        utxo: UTXO,
        keyPair: BIP32Interface,
    ): PsbtInputExtended {
        const input: PsbtInputExtended = {
```

and add this somewhere on top

```ts
export interface PsbtInputExtended extends PsbtInput, TransactionInput {}
```

Reply:
```ts
import { PsbtInput } from 'bip174/src/lib/interfaces';
import { TransactionInput } from 'bitcoinjs-lib/src/psbt';

export interface PsbtInputExtended extends PsbtInput, TransactionInput {}
```